R/cv_varsel.R

```diff
@@ -609,26 +609,24 @@ loo_varsel <- function(refmodel, method, nterms_max, ndraws,
   # turn, i.e., brms's `reloo` argument) currently cannot be used because all
   # these techniques result in new MCMC draws for the reference model, meaning
   # that the projection would have to be adapted. Therefore, it is easier to
-  # recommend K-fold CV (for the reference model refits, i.e., brms's `reloo`
+  # recommend K-fold-CV (for the reference model refits, i.e., brms's `reloo`
   # argument, another reason is that they can quickly become as costly as
-  # K-fold CV).
-  warn_pareto(
-    n07 = sum(pareto_k > 0.7), n05 = sum(0.7 >= pareto_k & pareto_k > 0.5),
-    warn_txt_start = paste0("In the calculation of the reference model's ",
-                            "PSIS-LOO CV weights, "),
-    warn_txt_mid_common = paste0(" (out of ", n, ") Pareto k-values are "),
-    warn_txt_end = paste0(
-      ". Moment matching (see the loo package), mixture importance ",
-      "sampling (see the loo package), and `reloo`-ing (see the brms package) ",
+  # K-fold-CV).
+  warn_pareto(n07 = sum(pareto_k > 0.7), n = n,
```

Comment:
If I read this (and the definition of `warn_pareto()` correctly, then `n07` still refers to the fixed threshold of 0.7, but the `khat_threshold` printed by `%s` refers to the updated threshold. Shouldn't they be consistent? So `n07` would need to be updated as well?

Reply:
I guess this is a mistake, I'll check

Reply:
Fixed

Reply:
Thanks